src/ZODB/interfaces.py

```diff
@@ -280,9 +281,9 @@ class IStorageWrapper(Interface):
     - Out-of-band invalidation support
 
       A storage can notify it's wrapper of object invalidations that
```

Comment:
s/it's/its/

Reply:
Yes, but I'm not changing this line at all and the "it's" come from old text. There are other usage of "it's" in `ZODB/interfaces.py` where I would put "its" by myself. I thought it is kind of US style or something similar and not delved into changing what is semantically not covered by the patch subject. If we are to change "it's" -> "its" I think it is better to do in a separate patch and for the whole file or whole codebase.

Thanks for the approval.